src/workloads/sharding/WriteOneWithoutShardKeyUnshardedCollection.yml

```diff
@@ -0,0 +1,92 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  On an unsharded collection on a single shard cluster, run workloads that updateOne, deleteOne, and findAndModify.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+  Database: &Database test
+  DocumentCount: &DocumentCount 10000  # Number of documents to insert and modify.
+
+Actors:
+- Name: CreateShardedCluster
+  Type: AdminCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationMetricsName: EnableSharding
+      OperationName: AdminCommand
+      OperationCommand:
+        enableSharding: *Database
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: LoadInitialData
+  Type: MonotonicSingleLoader
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    BatchSize: 1000
+    DocumentCount: *DocumentCount
+    Database: *Database
+    Document: {x: 1, y: 1, z: 1}
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: CRUDOpsWithoutShardKey
+  Type: CrudActor
+  Threads: 1  # Use 1 thread to avoid updates throwing WriteConflict errors.
```

Comment:
If you did want more concurrent threads, you might be able to target separate docs that are filtered on ActorId or an expression thereof like [here](https://github.com/mongodb/genny/blob/58901acf3ed1a4dd41f55f41f1d46689ff804e51/src/workloads/docs/Generators.yml#L126), I'm pretty sure those are actor-thread unique. Up to you on that!

Reply:
Thanks for the suggestion. Jason is OOO right now, so just waiting on his final approval but good to know that the genny syntax is looking good.